### PR TITLE
Remove duplicate text for Remote derive address

### PR DESCRIPTION
**Pull request by Maar-io**

### CODE REVIEW

File: docs/xcm/building-with-xcm/xc-remote-transact.md [hunk 1]
```diff
@@ -121,20 +123,6 @@ This means that the following instructions like `WithdrawAsset` and `Transact` w
...
-| Name | Address |
-| ---- | ------- |
-| Contract H160 Address | `0x48DD0a20a199f96B56eCE7e994D83614A148aA63` |
-| Contract Derived SS58 Address | `agn53DdEuRgQsvgxqj5M1AecxB6LpbXT7T1R1hjVcoEBR6M` |
-| SS58 Address Public Key | `0xd219fe1b02545c7dd7e718b1530b4e32b23288351f61e5975c7dc49b004ff119` |
-| Caller Multilocation | `{ parents: 1, interior: X2 ( Parachain(2000), AccountId32 {network: NetworkId::Any, id: 0xd219f...f119 } ) }` |
-| Derived Account32Hash Address | `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP` |
-
-This means that the instructions like `WithdrawAsset` and `Transact` will be executed on remote chain as if origin was the address `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP`.
-
```
Comment (Dinonard) [inline docs/xcm/building-with-xcm/xc-remote-transact.md:137]: This one should be kept as it was.

File: docs/xcm/building-with-xcm/xc-remote-transact.md [hunk 2]
```diff
@@ -101,6 +101,8 @@ To simplify the API via which EVM smart contracts send the `Transact` instructio
 
 `DescendOrigin` will ensure that the origin is correctly set to be the smart contract's derived ss58 address.
 
+### Remotely Derived Contract Address
```
Comment (Dinonard) [inline docs/xcm/building-with-xcm/xc-remote-transact.md:104]: You should remove content from line 104 up to line 116.

**Review by Dinonard — CHANGES_REQUESTED** [review]

File: docs/xcm/building-with-xcm/xc-remote-transact.md [hunk 3]
```diff
@@ -133,7 +121,7 @@ For example, let's assume you have a contract deployed on Shibuya and are callin
 | Caller Multilocation | `{ parents: 1, interior: X2 ( Parachain(2000), AccountId32 {network: NetworkId::Any, id: 0xd219f...f119 } ) }` |
 | Derived Account32Hash Address | `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP` |
 
-This means that the instructions like `WithdrawAsset` and `Transact` will be executed on remote chain as if origin was the address `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP`.
+This means that the following instructions like `WithdrawAsset` and `Transact` will be executed as if origin was address `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP`.
```
Comment (Dinonard) [inline docs/xcm/building-with-xcm/xc-remote-transact.md:124]: They're not _following_, they have been covered in the previous chapter.

Comment (Maar-io) [reply docs/xcm/building-with-xcm/xc-remote-transact.md]: done

**Review by Dinonard — APPROVED** [review]
Great, thanks!

**Pull request merged**